core/src/main/java/hudson/console/ConsoleNote.java

```diff
@@ -270,10 +270,8 @@ public static ConsoleNote readFrom(DataInputStream in) throws IOException, Class
                 }
             }
 
-            Jenkins jenkins = Jenkins.get();
```

Comment:
Might be a regression after the previous changes. Looks like the null check below was quite deliberate

Reply:
I replaced the get call with the getInstanceOrNull and kept the null change.